docs/lean_inception/dia_3.md

```diff
@@ -1 +1,49 @@
-TO-DO
\ No newline at end of file
+# Lean Inception - Dia 3
+
+## Histórico de versões
+
+| Versão | Data       | Descrição | Autores |
+| ------ | ---------- | --------- | ------- |
+| 1.0    | 13/04/2024 | Abertura do documento | [Gustavo Abrantes ](https://github.com/GustaaSZ) |
+| 1.1    | 19/04/2024 | Imagem Brainstorm e Revisão Técnica | [Gustavo Abrantes ](https://github.com/GustaaSZ) e [Yuri Ezequie ](https://github.com/Yuri57Ezequiel)|
+
+## Sobre
+
+O terceiro dia de uma Lean Inception é uma etapa crítica do processo, pois envolve atividades fundamentais para o desenvolvimento do produto. Neste dia (10/04/2024), a equipe responsável pelo desenvolvimento (EPS/MDS) juntamente com os clientes do projeto (Enactus), se concentram em uma tarefa fundamental: o Brainstorm de Funcionalidade.
+
+No final dele, a equipe deve ter uma lista refinada de funcionalidades priorizadas e bem fundamentadas para o produto. Isso ajuda a garantir que o desenvolvimento se concentre nas áreas mais importantes e que todos na equipe tenham uma compreensão compartilhada das metas do projeto.
+
+Como em reuniões anteriores, as equipes haviam tido um diálogo inicial sobre o possível rumo do produto final (se ele continuaria sendo um produto web, ou se passaria a ser um app nativo), no dia 10/04/2024, não foi possível concluir totalmente o Brainstorm de Funcionalidades, apenas de forma parcial. Desta forma, a equipe de EPS/MDS ficou responsável pelo término da clusterização até dia 11/04 ao 12h e a equipe da Enactus responsável pela validação desta clusterização até o prazo máximo do dia 15/04 ao 12h de forma assíncrona.
+
+## *Brainstorm* de Funcionalidades
+Um brainstorm de funcionalidades é uma atividade em grupo onde as equipes geram ideias e conceitos para as funcionalidades ou recursos em um produto ou projeto, de maneira colaborativa e sem críticas iniciais. objetivo é criar um conjunto inicial de ideias que possam ser posteriormente avaliadas e priorizadas.
+
+### *Brainstorm* do GEROcuidado
+
+![Brainstorm](../assets/Brainstorm_funcionalidades.png)
+
+## Revisão técnica, de negócio e UX
```

Comment:
Acho que pode deixar isso no lean inception dia 4, já que nao terminamos de fazer ainda.
Pra vcs não perderem o que já fizeram só pegar essa parte e colocar no lean_inception/dia_4.md pra já ficar lá!

Reply:
Ok, porém a parte de revisão técnica, de negócio e UX ficaria no dia 4 com o sequenciador e o canvas mvp?

Reply:
Na vdd, to querendo criar um dia 5 KKKK pra colocar o sequenciador e o canvas, mas vamos ver!
Pode deixar só isso lá por enquanto

Reply:
Inclusive, já ta assim no nosso kick off
![image](https://github.com/fga-eps-mds/2024-1-GEROcuidado-Doc/assets/44625056/f80d4a6e-e13e-4e65-a831-102495076609)
@GustaaSZ

Reply:
Perfeito! Coloquei a parte de revisãi téc. e ux no dia 4, porém não dei commit e nem criei uma nova branch para o dia 4, deixa assim mesmo, ou já cria uma nova branch e commita?

Reply:
Pode commitar nessa branch msm `docs/documentation_dia3_lean_Inception`, que dai ja fica la certinho